test/strict.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+
+var fs = require('fs')
+var glob = require('glob')
+var path = require('path')
+var test = require('tap').test
+
+test('all JavaScript source files use strict mode', function (t) {
+  var globStr = '**/*.js'
+  var root = path.resolve(__dirname, '../')
+  glob(globStr, {
+    cwd: root,
+    ignore: 'node_modules/**/*.js'
+  }, function (err, files) {
+    if (err) { throw err }
+    var line = '\'use strict\'\n'
```

Comment:
Is it nicer (more readable) if this line is surrounded by double quotes instead of single quotes? It feels better on my brain.